spanner/api/Spanner.Samples/InsertStructSampleDataAsync.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_write_data_for_struct_queries]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+public class InsertStructSampleDataAsyncSample
+{
+    public async Task<int> InsertStructSampleDataAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+        List<Singer> singers = new List<Singer> {
+            new Singer {SingerId = 6, FirstName = "Elena", LastName = "Campbell"},
+            new Singer {SingerId = 7, FirstName = "Gabriel", LastName = "Wright"},
+            new Singer {SingerId = 8, FirstName = "Benjamin", LastName = "Martinez"},
+            new Singer {SingerId = 9, FirstName = "Hannah", LastName = "Harris"},
+            new Singer {SingerId = 11, FirstName = "Anthony", LastName = "Hunter"},
+            new Singer {SingerId = 12, FirstName = "Earlean", LastName = "Holland"},
```

Comment:
I'm curious why these two addition rows were added and the reason for the non-consecutive primary key.

Reply:
Need more data for test  the samples. I will make sure that it should be consecutive. Thanks!!

Reply:
I've left similar comments in other samples, but I think we should be receiving the list of singers as a parameters, and initialize that on tests. In my opinion this just adds clutter to the sample for someone who should be familiar with .NET (knows how to initialize objects and lists) and just want to know how to insert something in Spanner.

Reply:
Also, if we remove the initialization from all the insert samples we might only need one sample to show in docs.

Reply:
We can add singers as parameter but still we need it in initialization for query the data.

Reply:
If this sample is part of the Getting Started guide, then it can remain as is. Else data initialization should be done as part of running tests, either in the fixture or in individual tests.